### PR TITLE
Fix ftbfs on armXX

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

**Comment by AlanGriffiths** [comment]
Fixes the arm build:
```plain
Build status as of 2021-03-02 15:42:51.874409:
        arch=arm64      state=Successfully built
        arch=armhf      state=Successfully built
        arch=amd64      state=Successfully built
```

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -16,13 +16,12 @@ architectures:
...
   SHELL: bash
   LC_ALL: C.UTF-8
-  LD_LIBRARY_PATH: ${LD_LIBRARY_PATH}:${SNAP}/usr/lib/:${SNAP}/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/:/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/pulseaudio:/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/samba
+  LD_LIBRARY_PATH: ${SNAP}/usr/lib/:${SNAP}/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/:/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/pulseaudio:/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/samba
   PATH: $SNAP/bin/:$SNAP/usr/bin/:${PATH}
   # XDG config
   XDG_CACHE_HOME:  $SNAP_USER_COMMON/.cache
   XDG_CONFIG_HOME: $SNAP_USER_DATA/.config
   WINDOWING: wayland
-  WAYLAND_DISPLAY: wayland-0
```
Comment (Saviq) [inline snap/snapcraft.yaml:25]: This was required for Kodi 19 to work on mir-kiosk here.

Do we need a `WAYLAND_DISPLAY=${WAYLAND:-wayland-0}` somewhere, then?

**Review by Saviq — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (AlanGriffiths) [reply snap/snapcraft.yaml:25]: > Do we need a `WAYLAND_DISPLAY=${WAYLAND:-wayland-0}` somewhere, then?

Yes. Seems like Kodi is using it to detect a Wayland environment. Sigh!

**Review by Saviq — APPROVED** [review]
Yup.

bors merge

**Comment by Saviq** [comment]
bors retry

**Pull request merged**